composer.json

```diff
@@ -102,6 +102,9 @@
     "sort-packages": true,
     "allow-plugins": {
       "symfony/flex": true
+    },
+    "platform": {
+      "php": "7.2.5"
```

Comment:
Nice addition!

Also, feel free to add `"php": "^7.2",` as a dependency. As we do in EngineBlock: 

https://github.com/OpenConext/OpenConext-profile/blob/a3aa579fa1565bb76c3693e246c18e8e572e8b7a/composer.json#L14

But feel free to leave this as-is. For now this is good enough.

Reply:
I will leave it for now.